js/index.js

```diff
@@ -0,0 +1,325 @@
+//Function to validate form
+function validateForm() {
+    var productid = document.getElementById("productid").value;
+    var productname = document.getElementById("productname").value;
+    var price = document.getElementById("price").value;
+    var description = document.getElementById("description").value;
+    var regex = /^[^\s][a-zA-Z]+[a-zA-Z0-9 ]*[^\s]$/;
+
+    if (productid == "" || productname == "" || price == "" || description == "") {
+        alert("All fields are required!");
+        return false;
+    }
+    else if (isNaN(productid)) {
+        alert("Only numeric Value");
```

Comment:
Manage proper style of text : Only numeric Value to **Only numeric value**

Reply:
done sir.